packages/trader/src/index.ts

```diff
@@ -92,7 +91,11 @@ export async function run(logger: winston.Logger, web3: Web3): Promise<void> {
       await retry(
         async () => {
           // Update the price feeds & gasEstimator.
-          await Promise.all([tokenPriceFeed.update(), referencePriceFeed.update(), gasEstimator.update()]);
+          await Promise.all([
+            (tokenPriceFeed as any).update(),
```

Comment:
@mrice32 one slightly painful thing is that the type returned by the `createTokenPriceFeedForFinancialContract` does not return an appropriate type so I had to type cast this. do you see any better solution to this?